apps/web/env.mjs

```diff
@@ -0,0 +1,110 @@
+import { createEnv } from "@t3-oss/env-nextjs";
+import { z } from "zod";
+
+export const env = createEnv({
+  /*
+   * Serverside Environment variables, not available on the client.
+   * Will throw if you access these variables on the client.
+   */
+  server: {
+    DATABASE_URL: z.string().url(),
+    PRISMA_GENERATE_DATAPROXY: z.enum(["1", "0", ""]).optional(),
+    NEXTAUTH_SECRET: z.string().min(1),
```

Comment:
Hey @mattinannt 
and specific reason of setting
NEXTAUTH_SECRET and DATABASE_URL values as required?
As i was not able to start application because of required validation.

Reply:
@Shubhdeep12 what would you like to do with Formbricks without the database url? 😉
I made them required because the database url is required due to the need for a database connection and the nextauth secret is required for a secure nextauth session.

Reply:
Yes, it makes sense.
But my concern was the failure of the command(pnpm dev --filter=web...) while running formbricks locally - it was failing because these were set as required, the moment I changed my local code and add .optional() for both, I was able to run the app.

Reply:
We are currently investigating this issue. It seems that the reference to the .env file in the project root is not correctly recognized and the error is thrown even though the variables are set.